docs/en/writing-running-appium/caps.md

```diff
@@ -60,10 +60,10 @@ These Capabilities are available only on Android-based drivers (like
 
 |Capability|Description|Values|
 |----|-----------|-------|
-|`appActivity`| Activity name for the Android activity you want to launch from your package. This often needs to be preceded by a `.` (e.g., `.MainActivity` instead of `MainActivity`)|`MainActivity`, `.Settings`|
-|`appPackage`| Java package of the Android app you want to run|`com.example.android.myApp`, `com.android.settings`|
-|`appWaitActivity`| Activity name/names, comma separated, for the Android activity you want to wait for|`SplashActivity`, `SplashActivity,OtherActivity`, `*`, `*.SplashActivity`|
-|`appWaitPackage`| Java package of the Android app you want to wait for|`com.example.android.myApp`, `com.android.settings`|
+|`appActivity`| Activity name for the Android activity you want to launch from your package. This often needs to be preceded by a `.` (e.g., `.MainActivity` instead of `MainActivity`). By default this capability is received from the package manifest (action: android.intent.action.MAIN , category: android.intent.category.LAUNCHER) |`MainActivity`, `.Settings`|
+|`appPackage`| Java package of the Android app you want to run. By default this capability is received from the package manifest (@package attribute value)|`com.example.android.myApp`, `com.android.settings`|
+|`appWaitActivity`| Activity name/names, comma separated, for the Android activity you want to wait for. By default the value of this capability is the same as for `appActivity`. You must set `appWaitActivity` to wait for activities if your capability has `appActivity` and `appPackage` |`SplashActivity`, `SplashActivity,OtherActivity`, `*`, `*.SplashActivity`|
```

Comment:
In this line, I'd like to leave:

1. the capability has only `app`
    - Then https://github.com/appium/appium-adb/blob/master/lib/tools/apk-utils.js#L120 will be called since `waitActivity` will be `appActivity` by default.
2. the capability has `appActivity` and `appPackage` additionally
    - Then https://github.com/appium/appium-adb/blob/master/lib/tools/apk-utils.js#L120 will be skipped unless we specify `appWaitActivity` by manual.

(The behaviour is same as uiautomator2 and selendroid driver as well)